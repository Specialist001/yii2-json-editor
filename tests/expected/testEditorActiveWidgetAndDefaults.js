modelmockDataJsonEditor_cb242086 = new JSONEditor(document.getElementById('modelmock-data-json-editor'), {"mode":"tree","onChange":function() {jQuery('#modelmock-data').val(modelmockDataJsonEditor_cb242086.getText());},"onModeChange":function(newMode, oldMode) {if (["tree"].indexOf(newMode) !== -1) {modelmockDataJsonEditor_cb242086.expandAll();}}}, {});
jQuery('#modelmock-data').parents('form').submit(function() {jQuery('#modelmock-data').val(modelmockDataJsonEditor_cb242086.getText());});
modelmockDataJsonEditor_cb242086.expandAll();