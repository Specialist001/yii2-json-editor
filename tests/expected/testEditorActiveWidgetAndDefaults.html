<form id="data-form" action="test" method="post"><div class="form-group field-modelmock-data">
<label class="control-label" for="modelmock-data">Data</label>
<input type="hidden" id="modelmock-data" name="ModelMock[data]" data-json-editor-name="modelmockDataJsonEditor_cb242086"><div id="modelmock-data-json-editor"></div>

<div class="help-block"></div>
</div></form>